### ISSUE TITLE
fix link to Combinator & UQDS in svn-dev document

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5545]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)| @keturn reported|
|-|-|
|Trac ID|trac#5545|
|Type|defect|
|Created|2012-03-13 20:54:32Z|
|Branch|https://github.com/twisted/twisted/tree/no-divmod-wiki-5545|

doc/core/development/policy/svn-dev.xhtml points to divmod wiki for Combinator and UQDS.  Those links don't work anymore.

<details><summary>Searchable metadata</summary>

```
trac-id__5545 5545
type__defect defect
reporter__acapnotic acapnotic
priority__normal normal
milestone__ 
branch__branches_no_divmod_wiki_5545 branches/no-divmod-wiki-5545
branch_author__acapnotic acapnotic
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1331672072000000 1331672072000000
changetime__1331704591000000 1331704591000000
version__None None
owner__acapnotic acapnotic

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)<a name="note_1"></a>|@keturn commented|
|-|-|

UQDS seems to be at http://twistedmatrix.com/trac/wiki/UltimateQualityDevelopmentSystem
That document's links to Combinator point to https://launchpad.net/divmod.org
I do not see a download link labeled "combinator."  There is nothing in https://launchpad.net/divmod.org/+download

You can browse it at https://bazaar.launchpad.net/~divmod-dev/divmod.org/trunk/files/head:/Combinator/ but that's not really helpful.

So there are sort of two bugs here: One is that the link is old and broken, the other is that there is no un-broken link for Combinator.

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)<a name="note_2"></a>|@keturn commented|
|-|-|

idnar points out it is actually possible to get a tarball out of launchpad, at something like https://bazaar.launchpad.net/~divmod-dev/divmod.org/trunk/tarball/2688?start_revid=2688

which is kinda gross, as you get the whole divmod repository and not just combinator, and it has a funny name.  But it doesn't require you to have bzr.

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)|@keturn set owner to @keturn<br>@keturn set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)<a name="note_4"></a>|@keturn commented|
|-|-|

(In [33762]) Branching to 'no-divmod-wiki-5545'

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)<a name="note_5"></a>|@keturn removed owner<br>@keturn set status to `new`|
|-|-|

and the fact that this branch was created suggests that I have combinator working.

**Comment by twisted-trac** [comment]
|[<img alt="MostAwesomeDude's avatar" src="https://avatars.githubusercontent.com/u/118035?s=50" width="50" height="50">](https://github.com/MostAwesomeDude)<a name="note_6"></a>|@MostAwesomeDude set owner to @keturn|
|-|-|

Well, pointing these to **anywhere** that we can control and edit is a good idea. Making it the tm.com wiki seems like a really great plan. And, as the icing, those pages already exist and have relevant content. +1, please merge. :3

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)<a name="note_7"></a>|@keturn set status to `closed`|
|-|-|

(In [33793]) Merge no-divmod-wiki-5545: update old svn-dev links to use tm.com wiki

Author: acapnotic
Reviewers: MostAwesomeDude
Fixes: [#5545](https://github.com/twisted/twisted/issues/5545)
